app/services/deploy_service.rb

```diff
@@ -61,7 +61,7 @@ def create_pull_request
       deploy_strategy.arguments['base'],
       deploy_strategy.arguments['head'],
       'Deploy',
-      'This is an automatically generated release PR!'
+      "This is an automatically generated release PR!\n\nHumans: remember to _merge_ (not squash) this PR."
```

Comment:
Should we update this to "(not squash or rebase)"? I believe we run into the same problem with rebase-merged deploy PRs.

Reply:
I didn't realize that--sure!